hordak/tests/test_worked_examples.py

```diff
@@ -155,7 +155,7 @@ def test_utility_bill(self):
 
         # We receive the actual bill (we are moving a negative amount of money,
         # as this is an outgoing)
-        self.gas_payable.transfer_to(self.cash, Money(-300, "EUR"))
+        self.cash.transfer_to(self.gas_payable, Money(300, "EUR"))
```

Comment:
This is incredible! I'm insanely excited to see this change right here.

I'm sure you already know this, but just stating it explicitly to be on the same page: 

This is a breaking change for anyone doing RHS -> LHS, vice versa - using the current `.transfer_to`. Saw that `accounting_transfer_to` "effectively" replaces it which I think is 100% correct, just it's a significant change and warrants an "Upgrade to 2.0" document.

Reply:
I'm very pleased to hear it! And yes, absolutely understood re upgrading. I've [put these notes in the changelog](https://github.com/adamcharnock/django-hordak/blob/experiment/amount-to-debit-credit/CHANGES.txt). How does that seem to you?